conf/init.properties.example

```diff
@@ -1,6 +1,12 @@
-DRILL_HOST=http://localhost:8047
+DRILL_HOST=http://localhost
+DRILL_PORT=8047
+DRILL_VERSION=1.15.0-SNAPSHOT
 DRIVER_TYPE=CHROME
 SECURE_DRILL=true
 DEFAULT_TIMEOUT=2
-DRILL_USER_NAME=mapr
-DRILL_USER_PASSWORD=mapr
\ No newline at end of file
+DRILL_USER_NAME=mapr1
+DRILL_USER_PASSWORD=mapr
+DRILL_SECOND_USER_NAME=mapr2
+DRILL_SECOND_USER_PASSWORD=mapr
+DRILL_ADMIN_USER_NAME=mapr
+DRILL_ADMIN_USER_PASSWORD=mapr
```

Comment:
If we need to run tests on secured Drill, only one user is required. For the security tests it's better to define users in tests themselves. It can be hard-coded by now, but later they should be moved to resources.

Reply:
done